### PR TITLE
7501 - Guides text changes

### LINKED ISSUE #7501: Guides Text Updates
We need to make changes to the Guides text to reflect proper usage of the term "Dataverse"

### PR DESCRIPTION
**Pull request by djbrooke**
**What this PR does / why we need it**:

Updates of the Dataverse Software Guides based on Merce's post in the Google Group:

https://groups.google.com/g/dataverse-community/c/pD-aFrpXMPo

**Which issue(s) this PR closes**:

Closes #7501 

**Special notes for your reviewer**:

It's a big PR! Hopefully I got all the spots. Happy to discuss with whoever picks this up.

I still have some image edits coming from @dliburd but the text should be ready to be reviewed.

**Suggestions on how to test this**:

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

**Is there a release notes update needed for this change?**:

Yes, I think so. Included.

**Additional documentation**:

### CODE REVIEW

File: doc/sphinx-guides/source/developers/dev-environment.rst [hunk 1]
```diff
@@ -36,29 +36,29 @@ Install Netbeans or Maven
...
+Clone the Dataverse Software Git Repo
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Fork https://github.com/IQSS/dataverse and then clone your fork like this:
 
 ``git clone git@github.com:[YOUR GITHUB USERNAME]/dataverse.git``
 
-Build the Dataverse War File
+Build the Dataverse Software War File
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (mheppler) [inline doc/sphinx-guides/source/developers/dev-environment.rst:54]: Formatting requires additional header characters for this line to match the header text length above.

File: doc/sphinx-guides/source/developers/tools.rst [hunk 2]
```diff
@@ -237,7 +237,7 @@ The script above will run until you stop it, and will output something like:
...
...g generation) garbage collections, and in the old generation allocation being almost 70% full. These numbers in the example are clearly quite high and are an indication of some problematic memory allocation by the Dataverse installation page - if this is the result of something you have added to the page, you probably would want to investigate and fix it. However, overly generous memory use **is not the same as a leak** necessarily. What you want to see now is how much of this allocation can be reclaimed by "Full GC". If all of it gets freed by ``FGC``, it is not the end of the world (even though you do not want your system to spend too much time running ``FGC``; it costs CPU cycles, and actually freezes the application while it's in progress!). It is however a **really** serious problem, if you determine that a growing portion of the old. gen. memory (``"O"`` in the ``jmap`` output) is not getting freed, even by ``FGC``. This *is* a real leak now, i.e. something is leaving behind some objects that are still referenced and thus off limits to garbage collector. So look for the lines where the ``FGC`` counter is incremented. For example, the first ``FGC`` in the example output above:
```
Comment (mheppler) [inline doc/sphinx-guides/source/developers/tools.rst:240]: This page capitalizes "Dataset" and "Datafile" a lot. Don't think it's necessary. Worth reviewing.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 3]
```diff
@@ -516,7 +510,7 @@ edited:**
 
 2. ``hostname="localhost:8080";``
 
-   changed to point to the dataverse app, from which TwoRavens will be obtaining the metadata and data files. (don't forget to change 8080 to the correct port number!)
+   changed to point to the Dataverse installation app, from which TwoRavens will be obtaining the metadata and data files. (don't forget to change 8080 to the correct port number!)
```
Comment (mheppler) [inline doc/sphinx-guides/source/installation/r-rapache-tworavens.rst:513]: Not sure is "app" is necessary here with the addition of "installation".

File: doc/sphinx-guides/source/user/dataset-management.rst [hunk 4]
```diff
@@ -99,23 +99,23 @@ rsync is typically used for synchronizing files and directories between two diff
 File Upload Script
 ~~~~~~~~~~~~~~~~~~
 
-An rsync-enabled Dataverse installation has a file upload process that differs from the traditional browser-based upload process you may be used to. In order to transfer your data to Dataverse's storage, you will need to complete the following steps:
+An rsync-enabled Dataverse installation has a file upload process that differs from the traditional browser-based upload process you may be used to. In order to transfer your data the to Dataverse installation's storage, you will need to complete the following steps:
```
Comment (mheppler) [inline doc/sphinx-guides/source/user/dataset-management.rst:102]: Misplaced "the", should go after the "to" not before.

File: doc/sphinx-guides/source/versions.rst [hunk 5]
```diff
@@ -1,10 +1,10 @@
 :orphan:
 .. _guides_versions:
 
-Dataverse Documentation Versions
+Dataverse Software Documentation Versions
 ================================
```
Comment (mheppler) [inline doc/sphinx-guides/source/versions.rst:5]: Additional header format characters required to match the length of the header text above.

**Review by mheppler — CHANGES_REQUESTED** [review]
Did not build the branch locally to see how the changes build in Sphinx. Happy to do so after these initial fix requests/suggestions are resolved.

Also, I suggest reviewing the capitalization of "Dataverse Software", as I have found many variations:

- Dataverse software
- Dataverse Software
- Dataverse 5.0 software
- Dataverse Software 4.6.1
- Dataverse software 4.8.1
- Dataverse software version 4.7
- Dataverse software v.4.6.1

File: doc/sphinx-guides/source/versions.rst (hunk 5 above)
Comment (djbrooke) [reply doc/sphinx-guides/source/versions.rst]: fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc

File: doc/sphinx-guides/source/user/dataset-management.rst (hunk 4 above)
Comment (djbrooke) [reply doc/sphinx-guides/source/user/dataset-management.rst]: fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst (hunk 3 above)
Comment (djbrooke) [reply doc/sphinx-guides/source/installation/r-rapache-tworavens.rst]: fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc

File: doc/sphinx-guides/source/developers/dev-environment.rst (hunk 1 above)
Comment (djbrooke) [reply doc/sphinx-guides/source/developers/dev-environment.rst]: fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc

File: doc/sphinx-guides/source/api/intro.rst [hunk 6]
```diff
@@ -224,11 +224,11 @@ See :doc:`faq`.
...
 ------------
 
-Dataverse API questions are on topic in all the usual places:
+Dataverse software API questions are on topic in all the usual places:
 
 - The dataverse-community Google Group: https://groups.google.com/forum/#!forum/dataverse-community
-- Dataverse community calls: https://dataverse.org/community-calls
-- The Dataverse chat room: http://chat.dataverse.org 
-- The Dataverse ticketing system: support@dataverse.org
+- Dataverse Project community calls: https://dataverse.org/community-calls
```
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/sphinx-guides/source/api/intro.rst]: fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc

File: doc/sphinx-guides/source/api/intro.rst [hunk 7]
```diff
@@ -104,53 +104,53 @@ Power users may be researchers or curators who are comfortable with automating p
...
 - The :doc:`/api/external-tools` section this guide.
 - The :doc:`apps` section of this guide.
 
-|Start| Good starting points are the three sections above to get a sense of third-party software that already integrates with Dataverse, followed by the :doc:`getting-started` section.
+|Start| Good starting points are the three sections above to get a sense of third-party software that already integrates with the Dataverse software, followed by the :doc:`getting-started` section.
 
-Developers of Dataverse API Client Libraries 
-++++++++++++++++++++++++++++++++++++++++++++
+Developers of Dataverse software API Client Libraries 
++++++++++++++++++++++++++++++++++++++++++++++++++++++
```
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/sphinx-guides/source/api/intro.rst:144]: fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc

File: doc/sphinx-guides/source/admin/reporting-tools-and-queries.rst [hunk 8]
```diff
@@ -10,10 +10,10 @@ Reporting tools and queries created by members of the Dataverse community.
...
 
-* Useful queries from the Dataverse Community (<https://docs.google.com/document/d/1-Y_iUduSxdDNeK1yiGUxe7t-Md7Fy965jp4o4m1XEoE/edit#heading=h.avuoo5kf0mdt>): A community-generated and maintained document of postgresql queries for getting information about users and dataverses/datasets/files in your Dataverse installation. If you are trying to find out some information from Dataverse, chances are that someone else has had the same questions and it's now listed in this document. If it's not listed, please feel free to add it to the document.
\ No newline at end of file
+* Useful queries from the Dataverse Community (<https://docs.google.com/document/d/1-Y_iUduSxdDNeK1yiGUxe7t-Md7Fy965jp4o4m1XEoE/edit#heading=h.avuoo5kf0mdt>): A community-generated and maintained document of postgresql queries for getting information about users and dataverse collections,datasets, and files in your Dataverse installation. If you are trying to find out some information from your Dataverse installation, chances are that someone else has had the same questions and it's now listed in this document. If it's not listed, please feel free to add it to the document.
```
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/sphinx-guides/source/admin/reporting-tools-and-queries.rst]: fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc

File: doc/sphinx-guides/source/admin/harvestserver.rst [hunk 9]
```diff
@@ -4,12 +4,11 @@ Managing Harvesting Server and Sets
 .. contents:: |toctitle|
   :local:
 
-Your Dataverse as an OAI server
+Your Dataverse Installation as an OAI server
 -------------------------------
```
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/sphinx-guides/source/admin/harvestserver.rst]: fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc

File: doc/sphinx-guides/source/api/sword.rst [hunk 10]
```diff
@@ -59,16 +59,16 @@ Differences in Dataverse 4 from DVN 3.x lead to a few minor backward incompatibl
...
 - Deaccessioning is no longer supported. An alternative will be developed at https://github.com/IQSS/dataverse/issues/778
 
-- The Service Document will show a single API Terms of Use rather than root level and dataverse level Deposit Terms of Use.
+- The Service Document will show a single API Terms of Use rather than root level and Dataverse collection level Deposit Terms of Use.
 
 New features as of v1.1
 -----------------------
 
-- Dataverse 4+ supports API tokens and requires them to be used for APIs instead of a username and password. In the ``curl`` examples below, you will see ``curl -u $API_TOKEN:`` showing that you should send your API token as the username and nothing as the password. For example, ``curl -u 54b143b5-d001-4254-afc0-a1c0f6a5b5a7:``. See also :ref:`sword-auth`.
+- The Dataverse Software supports API tokens and requires them to be used for APIs instead of a username and password. In the ``curl`` examples below, you will see ``curl -u $API_TOKEN:`` showing that you should send your API token as the username and nothing as the password. For example, ``curl -u 54b143b5-d001-4254-afc0-a1c0f6a5b5a7:``. See also :ref:`sword-auth`.
```
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/sphinx-guides/source/api/sword.rst:67]: capitalization made consistent in https://github.com/IQSS/dataverse/pull/7567/commits/91d36f67ff259870648be2cc3894b9c6af5683a2

File: doc/sphinx-guides/source/api/sword.rst [hunk 11]
```diff
@@ -59,16 +59,16 @@ Differences in Dataverse 4 from DVN 3.x lead to a few minor backward incompatibl
...
+- SWORD operations no longer require "admin" permission. In order to use any SWORD operation in DVN 3.x, you had to be an "admin" on a Dataverse collection (the container for your dataset) and similar rules were applied in Dataverse Software 4.4 and earlier (the ``EditDataverse`` permission was required). The SWORD API has now been fully integrated with the Dataverse Software permission model such that any action you have permission to perform in the GUI or "native" API you are able to perform via SWORD. This means that even a user with a "Contributor" role can operate on datasets via SWORD. Note that users with the "Contributor" role do not have the ``PublishDataset`` permission and will not be able publish their datasets via any mechanism, GUI or API.
```
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/sphinx-guides/source/api/sword.rst:69]: capitalization made consistent in https://github.com/IQSS/dataverse/pull/7567/commits/91d36f67ff259870648be2cc3894b9c6af5683a2

File: doc/sphinx-guides/source/api/sword.rst [hunk 12]
```diff
@@ -49,7 +49,7 @@ Backward incompatible changes
 
 For better security than in DVN 3.x, usernames and passwords are no longer accepted. The use of an API token is required.
 
-Differences in Dataverse 4 from DVN 3.x lead to a few minor backward incompatible changes in the Dataverse implementation of SWORD, which are listed below. Old ``v1`` URLs should continue to work but the ``Service Document`` will contain a deprecation warning and responses will contain ``v1.1`` URLs. See also :ref:`known-issues`.
+Differences in Dataverse software 4 from DVN 3.x lead to a few minor backward incompatible changes in the Dataverse software implementation of SWORD, which are listed below. Old ``v1`` URLs should continue to work but the ``Service Document`` will contain a deprecation warning and responses will contain ``v1.1`` URLs. See also :ref:`known-issues`.
```
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/sphinx-guides/source/api/sword.rst]: capitalization made consistent in https://github.com/IQSS/dataverse/pull/7567/commits/91d36f67ff259870648be2cc3894b9c6af5683a2

File: doc/sphinx-guides/source/developers/tools.rst (hunk 2 above)
Comment (djbrooke) [reply doc/sphinx-guides/source/developers/tools.rst:240]: I agree that we could revisit, but I'll leave this be for now as out of scope.

**Comment by djbrooke** [comment]
Thanks @mheppler for the review! I believe I responded to all feedback.

**Review by mheppler — APPROVED** [review]

**Comment by mheppler** [comment]
Pulled locally and built in Sphinx to make sure it succeeded without any formatting warnings. Confirmed formatting is good and approved.

**Pull request merged**